.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - '3.x'
```

Comment:
Given that 1.x and 2.x are still supported, I think it would be better to have the same CI setup in all branches rather than maintaining both.

Reply:
I agree. We might decide to stop the 1.x branch as I think there is no need for it anymore (we bump the min PHP deps so much that I don't see any reasons to not upgrade to 2.x).

Reply:
Thats true, but I talked to Fabien, that we start with `3.x`, will add support for the other branches later on...

Reply:
I think I said 2.x and 3.x :) I've just checked, that what I said ;)